add-ground-truth.py

```diff
@@ -0,0 +1,218 @@
+"""
+Convert a recording to "ground truth" format and add it to another recording for benchmarking.
+
+For instance, you can get relatively accurate poses with Orbbec Femto Mega/Bolt device and then compare those with OAK-D poses.
+"""
+
+import json
+import os
+import numpy as np
+
+def synchronizeRecordings(args):
+    import sys
+    import importlib
+
+    sys.argv = [
+        'sync-imus.py',
+        f'{os.path.join(args.ground_truth, "data.jsonl")}',
+        f'{os.path.join(args.data, "data.jsonl")}',
+        '--no_plot'
+    ]
+    offset = importlib.import_module("sync-imus").synchronizeImus()
```

Comment:
Does this require you run the script with the current working directory set to the repository root? Could maybe add some usage example beginning with `cd calibration-scripts` if that is the case.

Reply:
I checked and worked outside repository root also, but I agree that this is quite ugly. Required because cannot have '-' in filename when using `import`. One option would be to rename sync-imus.py, but I didn't like that either because then the naming would differ from `sync-videos.py` and `sync-imu-video.py`.

Reply:
I recall that in some case python translates kebab-case module names to snake_case, so you could also try `import sync_videos`. But maybe it was the case that they just don't work.

Reply:
I tried `import sync_imus` but it doesn't work: `ModuleNotFoundError: No module named 'sync_imus'`